bigtable/tests/admin_test.cc

```diff
@@ -0,0 +1,151 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <google/protobuf/text_format.h>
+#include <google/protobuf/util/message_differencer.h>
+#include <sstream>
+#include "bigtable/admin/admin_client.h"
+#include "bigtable/admin/table_admin.h"
+
+int main(int argc, char* argv[]) try {
```

Comment:
This isn't super extensible as a one-method test. I'd suggest breaking it into two methods you invoke from main, first one testing create+get+delete operations, and the second testing column family modification.

It would be nice if there were a comment with simple description of the test. Even "Integration test to create and delete some tables" is sufficient.

Reply:
Done.